test/e2e/framework/pod/wait.go

```diff
@@ -302,7 +301,7 @@ func WaitForMatchPodsCondition(c clientset.Interface, opts metav1.ListOptions, d
 				return fmt.Errorf("Unexpected error: %v", err)
 			}
 			if !done {
-				conditionNotMatch = append(conditionNotMatch, format.Pod(&pod))
+				conditionNotMatch = append(conditionNotMatch, fmt.Sprintf("%s_%s(%s)", pod.Name, pod.Namespace, pod.UID))
```

Comment:
just noting that this can be problematic, if for some reason the definition in pkg/kubelet/util/format changes!

at the minimum we should include the following comment over the line:

```
// This should match the format defined in kubernetes/pkg/kubelet/util/format.
```

Reply:
sure , we can trace this function with comment.
Thanks